src/anaconda/.devcontainer/apply_security_patches.sh

```
@@ -0,0 +1,58 @@
+        CONDA_VERSION=$(conda search "${packages_array[$i,0]}" -c "$channel_name" | \
+            grep -E '^[[:alnum:]]' | \
+            awk '{print $2}' | \
+            sort -V | \
+            uniq | \
+            tail -n 2 | \
+            head -n 1)
+        if [[ -z "$CONDA_VERSION" ]]; then
+            echo "No version for ${packages_array[$i,0]} found in conda channel."
+            CONDA_VERSION="0"
+        fi
+        GREATER_VERSION_B=$((echo ${REQUIRED_VERSION}; echo ${CONDA_VERSION}) | sort -V | tail -1)
```

Shellcheck (0.10.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)